lib/cli/args.js

```diff
@@ -415,11 +415,27 @@ function makeUpdateArgs (type) {
   ];
 }
 
+function makeRunArgs (type) {
+  return [
+    ...globalExtensionArgs,
+    [[type], {
+      type: 'str',
+      help: `Name of the ${type} to run a script from, for example: ` + type === DRIVER_TYPE ? DRIVER_EXAMPLE : PLUGIN_EXAMPLE,
```

Comment:
Break this into different lines for readability as above, I think our max length is around 100 chars